.gitlab-ci.yml

```diff
@@ -52,6 +52,53 @@ build:
     paths:
       - image.tar
 
+test:format:
+  stage: test
+  needs: []
+  image: alpine:3.13
+  before_script:
+    - apk update
+    - apk upgrade
+    - apk add --update bash git shfmt
+    - shfmt -version
+  script:
+    - /bin/bash
+    - SCRIPTS=$(find modules configs scripts -type f
+            -not -name "*.md"
+            -not -wholename "scripts/test/*")
```

Comment:
I haven't seen this syntax before. Does this work as expected? Opens `bash` and then runs the rest of the script?

Reply:
1. Works as expected
2. Surprised me too
3. We could stick to `sh`, but I like `Bash` more.

Reply:
No, this is very nice to know. I don't remember where, but we have used some weird hacks to run things in bash.

Actually, it kind of makes sense. It just runs a `bash` process, and then gitlab keeps piping to/from stdin/stdout.

Reply:
haha, yup. It actually makes kinda sense :)